include/ppx/scene/scene_material.h

```diff
@@ -0,0 +1,302 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ppx_scene_material_h
+#define ppx_scene_material_h
+
+#include "ppx/scene/scene_config.h"
+#include "ppx/grfx/grfx_image.h"
+
+#define PPX_MATERIAL_IDENT_ERROR    "ppx_material_ident:error"
+#define PPX_MATERIAL_IDENT_UNLIT    "ppx_material_ident:unlit"
+#define PPX_MATERIAL_IDENT_STANDARD "ppx_material_ident:standard"
+
+namespace ppx {
+namespace scene {
+
+// Image
+//
+// This class wraps a grfx::Image and a grfx::SampledImage objects to make
+// pathing access to the GPU pipelines easier.
+//
+// scene::Image objects can be shared between different scene::Texture objects.
+//
+// Corresponds to GLTF's image object.
+//
+class Image
+    : public grfx::NamedObjectTrait
+{
+public:
+    Image(
```

Comment:
Should we document that these objects "own" the pImage and pImageView and will destroy them in the destructor? Same for sampler

Reply:
Done.